common/src/main/java/gov/cms/ab2d/common/repository/CoverageRepository.java

```diff
@@ -22,4 +23,28 @@
 
     Optional<Coverage> findByContractAndBeneficiaryAndPartDMonth(Contract contract, Beneficiary bene, int month);
 
+
+
+    @Modifying
+    @Query(" DELETE FROM Coverage c  ")
+    int deleteAllInBulk();
+
+    @Modifying
+    @Query(" DELETE FROM Coverage c  " +
+            " WHERE c.partDMonth = :month  ")
+    int deleteInBulk(int month);
+
+    @Modifying
+    @Query(" DELETE FROM Coverage c  " +
+            " WHERE c.contract.id = :contractId  ")
+    int deleteInBulk(Long contractId);
```

Comment:
If I have an integer, say, `5`, then `deleteInBulk(5)` would _appear_ to fit either of these invocation, which might be confusing a bit. `deleteByMonth` and `deleteByContractId` would fix this.

Reply:
done